cashu/wallet/cli/cli.py

```diff
@@ -17,7 +17,7 @@
 from ...core.base import TokenV3
 from ...core.helpers import sum_proofs
 from ...core.settings import settings
-from ...nostr.nostr.client.client import NostrClient
+from nostr.client.client import NostrClient
```

Comment:
The imports must remain relative otherwise the library can't be imported by other applications. There might also be pip package issues when `nostr` is fetched from its repo.